UI/GameSettingsScreen.cpp

```diff
@@ -311,7 +319,7 @@ void GameSettingsScreen::CreateViews() {
 	PopupMultiChoice *beziersChoice = graphicsSettings->Add(new PopupMultiChoice(&g_Config.iSplineBezierQuality, gr->T("LowCurves", "Spline/Bezier curves quality"), quality, 0, ARRAY_SIZE(quality), gr->GetName(), screenManager()));
 	beziersChoice->OnChoice.Add([=](EventParams &e) {
 		if (g_Config.iSplineBezierQuality != 0) {
-			settingInfo_->Show(gr->T("LowCurves Tip", "This option will significantly improve/reduce the quality of rendered splines and bezier curves"), e.v);
+			settingInfo_->Show(gr->T("LowCurves Tip", "Improve/Reduce the quality of rendered splines and bezier curves"), e.v);
```

Comment:
Well, it could probably use A tip.  I couldn't have pointed to a video game's graphical effect and said "that must be a bezier" back when I was a teenager (depsite knowing vaguely what bezier curves were.)

-[Unknown]